frontend/src/app/model/comparison-cases-data.ts

```diff
@@ -0,0 +1,6 @@
+export class ComparisonCasesData {
```

Comment:
you are right, the class is never used.
In the older calls before I needed such a class for specifying the type of my rest response.
But now since we have the new structure, all the responses are of the ryp AreaResponse. I simply forgot that during implementation because I was used to the old structure xD
I will delete the class.